docs/en/setup/backend/backend-storage.md

```diff
@@ -54,6 +56,34 @@ storage:
     concurrentRequests: ${SW_STORAGE_ES_CONCURRENT_REQUESTS:2} # the number of concurrent requests
 ```
 
+### ElasticSearch 6 With Https SSL Encrypting communications.
+
+example: 
+
+```yaml
+storage:
+  elasticsearch:
+    # nameSpace: ${SW_NAMESPACE:""}
+    user: ${SW_ES_USER:""} # User needs to be set when Http Basic authentication is enabled
+    password: ${SW_ES_PASSWORD:""} # Password to be set when Http Basic authentication is enabled
+    clusterNodes: ${SW_STORAGE_ES_CLUSTER_NODES:localhost:443}
+    keyStorePath: ${SW_SW_STORAGE_ES_SSL_JKS_PATH:"../es_keystore.jks"}
```

Comment:
s/keyStore/trustStore/ ? Since we are client-side, `trustStore` may be a more reasonable name. What do you think about this?

Reply:
> @JaredTan95 Look like we need a document update at es storage section.

You mean this?

Reply:
No, I prefer to select `trustStorePath` instead of `keyStorePath`